utils/context_manger.py

```
@@ -0,0 +1,40 @@
+class UpdateContext():
+    def __init__(self, data: ContextData, status: bool):
+        pass
+
+    def update_cahe(self):
+        pass
+
+    def update_db(self):
+        pass
+
+    def update(self):
+        self.update_redis(self.data)
```

Pylint (3.3.7)

[error] Instance of 'UpdateContext' has no 'update_redis' member

(E1101)